### PR TITLE
Prague 2020 speakers

### PR DESCRIPTION
**Pull request by plaindocs**
Would love some eyes and finesse on this.

### CODE REVIEW

File: docs/_data/prague-2020-sessions.yaml [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+  slug: the-rocky-road-to-docops-chris-noonan
+  series: Write the Docs Prague
+  series_slug: the-rocky-road-to-docops-chris-noonan
+  year: 2020
+  speakers:
+  - name: Chris Noonan
+    slug: chris-noonan
+    twitter: 
+    website: https://www.linkedin.com/in/cnoonan/
+  abstract: "<p><strong>Abstract</strong><br />\nTechnology and process are invariably
```
Comment (mxsasha) [inline docs/_data/prague-2020-sessions.yaml]:
```suggestion
  abstract: "<p>Technology and process are invariably
```
(We should actually do that in pretalx and re-import)

File: docs/_data/prague-2020-sessions.yaml [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+
+    <li>separated assignments from solving strategies</li>
+
+    </ul>
+
+    </li>
+
+    <li>
+
+    <p>We debugged our docs continously before deployment by</p>
```
Comment (mxsasha) [inline docs/_data/prague-2020-sessions.yaml]: These `<p>` tags are superfluous and create line breaks. Don't know how easy it is to fix in their pretalx abstract?

File: docs/conf/prague/2020/news/announcing-speakers.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
+We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.
+
+`Buy your tickets here! <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_
+
```
Comment (mxsasha) [inline docs/conf/prague/2020/news/announcing-speakers.rst:34]:
```suggestion

Our Opportunity Grant program is still open until September 1st. If buying a ticket is difficult for you, please don't hesitate to apply to the grant program.
```

File: docs/_data/prague-2020-sessions.yaml (hunk 2 above)
Comment (plaindocs) [reply docs/_data/prague-2020-sessions.yaml]: Yeah, this is partly a problem with our CSS though.

**Comment by plaindocs** [comment]
Thanks @mxsasha

**Comment by thatdocslady** [comment]
Two small comments: 
1. The speaker "Jen" proabably should have a surname, and the bullet points look weird inside that abstract (not sure if it's the preview or the source). 
2. Some of the speaker names are misaligned next to their pictures. It drives me crazy but not sure if it'll drive everyone crazy ;)

Otherwise looks great, and OMG THESE TALKS <3

File: docs/conf/prague/2020/news/announcing-speakers.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Our Opportunity Grant program is still open until September 1st. If buying a ticket is difficult for you, please don't hesitate to apply to the grant program.
+Thanks to our sponsors
```
Comment (thatdocslady) [inline docs/conf/prague/2020/news/announcing-speakers.rst:39]:
```suggestion

**Meetup organizer discount**: As a thanks to our meetup organizers, we're happy to offer you a 50% discount on your tickets. Please email us at prague@writethedocs.org to receive registration instructions.

Thanks to our sponsors
```

**Review by thatdocslady — APPROVED** [review]

**Pull request merged**